src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.Cookies.cs

```diff
@@ -216,10 +216,15 @@ private string GetCookieValue(HttpRequestData request)
             return cookieHeaderValue;
         }
 
-        [Fact]
+        [ConditionalFact]
         [SkipOnPlatform(TestPlatforms.Browser, "CookieContainer is not supported on Browser")]
         public async Task GetAsync_SetCookieContainerAndCookieHeader_BothCookiesSent()
         {
+            if (UseVersion == HttpVersion30)
```

Comment:
This will work as well, but what I've seen, we usually override the failing test in the H3 implementation class and put the `ActiveIssue` attribute on that.